default.yml

```diff
@@ -168,3 +168,7 @@ RSpec/NamedSubject:
 
 RSpec/NestedGroups:
   Max: 6
+
+# https://docs.rubocop.org/en/latest/versioning/
+AllCops:
+  NewCops: enable
```

Comment:
One of the issues we've had is rubocop breaking builds for us on minor version releases.

I would suggest we set NewCops to disabled instead.
We would only introduce changes at each major version release then, reducing the amount of possible breakage.

Reply:
Yes, almost all ws-style updates are breaking. In our repos it's rare that we can merge the dependabot PR as is, we almost always have to run `rubocop -a` to be able to merge.

Introducing changes at major version releases makes sense. Will change this to `disable`.

@cabello what do you think?

Reply:
I think it would also help reduce breaking changes if we could lock the version of rubocop. Currently it's >= 0.79: https://github.com/wealthsimple/ws-style/blob/master/ws-style.gemspec#L24

Reply:
Without an active maintainer for ws-style we'd essentially be forever pinned to the version.
I'd suggest leaving it unpinned, so we're occasionally prompted to keep things updated :)